### ISSUE TITLE
SW Brutal assault  can't use from any position under Vengeance of Nagarythe

### ISSUE THREAD

**Issue by Kweedko** [WarEmu/WarBugs#12349]
Actual behavior:
SW Brutal assault can't use from any position under Vengeance of Nagarythe
Expected behavior
SW Brutal assault can use from any position under Vengeance of Nagarythe

Steps to reproduce the problem:
Use VoN try to hit mob in front with BA. - if you reed the skill descrition it say that you can use it from any position while you Vegenful. On live it worked that way.

Screenshots/Videos or archive.org evidences:

**Comment by Torquemadra** [comment]
It works fine, it doesnt light up

**Comment by Kweedko** [comment]
> It works fine, it doesnt light up. This should light up IF it indeed did light on live, it doesnt so I surmise it didn. Probably fixable in client but minor qol issue.

Yes it lited on live. And this is not just lite for client side this skill is unable to use but for server it's fine(so NB sequence not working properly cause of that). I bet the same thing with BAL on WH.

**Comment by Gunlingers** [comment]
The Problem lies not in it lighting up. On WH Bal and knockdown works as intendent. BaL and pistol wip are grayed out on action bar while not in right position, behind target, like if the wrong stance is applyed.

                                   !THE PROBLEM IS ONLY THERE IN PVE!

I haven't played my SW some time and wondered today, that something is wrong with Brutal Assault and Flanking Shoot. They are both permanently highlighed as ''useable''(coloured).
They should be both greyed out as long as the positional requirement is not given.

This occures only on Practice Targets, ''enemy'' NPC's that are orange aggro set.

And some Instance PVE Boss Mobs as for example:
Ard'da feat (GB middle last boss)
BS Karn the Vanquisher (middel path second last boss)
Cryps Necromancer Malsidius (right wing second boss)

On SW at these Bosses we can no longer visually see on the Castbar, if one these Skills are ready, 
because they look like they are always ready. but if pressed, the just TRY to start the GDC and half of
the animation.

For NB for example (what is not important anyway) the problem lies in the fact, that NB by default 
checks for a skill to be activ (useable) by checking if it is coloured. Since Brutal Assault and Flanking Shot are now permanently highlighted as useable, NB will always set that skill at the top of the activ 
skills. It will go on with that, until that skill has been uses and triggered his CD. But as sayed, The NB part is not important, since it is just an Addon to help Players with small experience.
In such Bossfights it will brake the NB skillsrotation then and block almost all NB skills at all.
